tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -2106,6 +2106,14 @@ def test_max_invalid_shapes_and_axis(device, array, axis, dtype, is_module):
         'input_rhs': ['random'],
         'is_module': [False],
     })
+    # Dtype combinations
+    + chainer.testing.product({
+        'shape': [(2, 3)],
+        'in_dtypes,out_dtype': _in_out_dtypes_arithmetic,
+        'input_lhs': ['random'],
+        'input_rhs': ['random'],
+        'is_module': [False],
+    })
```

Comment:
How about simply modifying this line https://github.com/chainer/chainer/pull/6858/files#diff-608e32a89d09e18207f3edcbee9b24d6R2121 instead of appending a new set of configurations (I also think it's better we do test `is_module=True`).

Reply:
Removed this parameterization.